spec/list-spec.coffee

```diff
@@ -88,6 +88,17 @@ describe 'apm list', ->
     listPackages [], ->
       expect(console.log.argsForCall[1][0]).toContain 'test-module@1.0.0 (disabled)'
 
+  it 'displays only enabled packages --enabled is called', ->
```

Comment:
Think you forgot a `when` here :)

Also, can you also copy over a different (enabled) package just to confirm that the output isn't always `(empty)`? :grinning: